### ISSUE TITLE
Build for SDK 20.08

### ISSUE THREAD

**Issue by gasinvein** [flathub/org.freedesktop.Sdk.Extension.dotnet#10]
Freedesktop SDK 20.08 is released, please build the extension for it.

**Comment by Pobega** [comment]
Waiting on review here (and it needs to be merged into the right branch, not as-is)
https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet/pull/12
